src/components/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useQueryParam, StringParam } from 'use-query-params';
+import {
+  SearchFormWrapper,
+  Title,
+  Form,
+  Prefix,
+  Input,
+} from './SearchForm.styles';
+
+import { CTAButton } from '../CTAButton/CTAButton.styles';
```

Comment:
Fyi: An advantage of using `index.js` files the way I show below is that you have a consistent outward-facing API. I mean in this cases you import from `CTAButton/CTAButton.styles`, but in most other cases you would expect to import from `CTAButton/CTAButton`. IMO it shouldn't matter whether the CTAButton component is a functional or styled component. That's an implementation detail.

So I usually have an index.js file (here it would be /CTAButton/index.js`) looking like this

```
export { default } from './MyComponent';

// or here

export { CTAButton as default } from '.CTAButton.styles';
```

Now you can always import via `import CTAButton from '../CTAButton'.

But I guess there are other opinions as well :-). So just for your information

Reply:
It didn't feel right when I imported from .styles. To stay consistent I hope it's okay to create 
CTAButton/CTAButton.js and do export { CTAButton as default } from '.CTAButton.styles';

Reply:
Sure, why not